mobile/yarn.lock

```diff
@@ -1339,9 +1339,9 @@
     "resolve-from" "^5.0.0"
     "sucrase" "^3.20.0"
 
-"@expo/ngrok-bin-win32-x64@2.3.40":
```

Comment:
Will the change from win32 to linux affect anything?
Otherwise LGTM

Reply:
Hm I'm not so sure, I think the yarn.lock was updated when I did npm i, so I think it should revert back to win32 when windows users do npm i?